### PR TITLE
IDE-2020 Remove Configure plugin sdks shortcuts in Liferay preference

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

File: common/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/pref/LiferayUIPreferencePage.java [hunk 1]
```diff
@@ -172,10 +141,8 @@ public void widgetSelected( SelectionEvent e )
     {
         public static String clear;
         public static String clearAllSettings;
-        public static String configureSDKs;
         public static String createNewLiferayRuntime;
```
Comment (AndyWu2015) [inline common/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/pref/LiferayUIPreferencePage.java]: Hi Greg what is about this line ? It is also not used in this class .

File: common/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/pref/LiferayUIPreferencePage.java [hunk 2]
```diff
@@ -172,10 +141,8 @@ public void widgetSelected( SelectionEvent e )
...
         public static String createNewLiferayServer;
```
Comment (AndyWu2015) [inline common/plugins/com.liferay.ide.ui/src/com/liferay/ide/ui/pref/LiferayUIPreferencePage.java]: also this line

**Comment by AndyWu2015** [comment]
And sorry I forget to delete the imports and if you agree to delete the two lines (I commented on) , I will do and delete imports and resend .

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Merged, thanks! See 55cb65e...a81aba3 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**